### ISSUE TITLE
No way to differentiate a cancelled report from done with no data?

### ISSUE THREAD

**Issue by johnkw** [amzn/selling-partner-api-models#393]
MWS had a `DONE_NO_DATA` state.  I had sort of assumed SPAPI would still have `DONE_NO_DATA`, but searching the documentation I couldn't find it.

But then I found this strange documentation on the `CANCELLED` state, from https://github.com/amzn/selling-partner-api-docs/blob/main/references/reports-api/reports_2020-09-04.md :
> The report was cancelled. There are two ways a report can be cancelled: an explicit cancellation request before the report starts processing, or an automatic cancellation if there is no data to return.

It sounds like it's now impossible to differentiate a truly cancelled report, from a successful report that just happens to return 0 rows of data.  If so, that's a bizarre and horrible regression from MWS to SPAPI.  But if so, it should really be very explicitly documented that `DONE_NO_DATA` has been removed, and it's now impossible to differentiate a cancelled report from done with no data.

I sure hope this wasn't intentional though, and the SPAPI can be fixed to restore the `DONE_NO_DATA` status.

**Comment by shreeharsh-a** [comment]
Thank you for bringing this up. We have noted & forwarded this to the relevant team for review & action.

`Note: SP API or docs related issues / troubleshooting support is managed by a different team. We can help if the issue is related to the content published on this repo.`

**Comment by jakeleventhal** [comment]
@shreeharsh-a why wouldnt this issue be open until the issue is actually resolved

**Comment by shreeharsh-a** [comment]
@jakeleventhal thanks for your question. Only issues related to the content published on this repo will be tracked to closure here. Anything else will need to be raised & tracked via [Developer Support](https://developer.amazonservices.com/support) cases.

**Comment by jakeleventhal** [comment]
@shreeharsh-a I understand but this is something that I and probably dozens of others have raised in support tickets going back years. Surely there must be some publicly tracked issue list?

**Comment by johnkw** [comment]
It's great that the SPAPI team has been informed of this bug report. However it would be helpful to leave the bug open until it's fixed, so people know where to look for information on this bug.

Closing bugs that are not fixed is extremely counterproductive.

We used to be able to search easily for the open bugs and when we hit them again could quickly refresh our memory on the status and what to try to do as a workaround.  Now they can still be found via Google or more complicated github searches.  But it's much easier to be able to search github.com for OPEN bugs when looking for bugs that are still actually supposed to be open because they're not fixed.
